binrz/man/rz-ax.1

```diff
@@ -66,13 +86,19 @@ Available variable types are:
   \-int  \->  hex    rz-ax \-77
   \-hex  \->  int    rz-ax 0xffffffb3
   int   \->  bin    rz-ax b30
+  int \-> ternary   rz-ax t42
+  ternary -> int   rz-ax 1010dt
```

Comment:
I think this should be aligned with others

Reply:
![image](https://github.com/rizinorg/rizin/assets/62786136/aef5e2f2-9043-4429-8658-0c352d4bbaa5)
have a look